### PR TITLE
Added isort for import order validation

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: course_discovery/apps/api/pagination.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
-from rest_framework.pagination import (
-    PageNumberPagination as BasePageNumberPagination,
-    LimitOffsetPagination,
-)
+from rest_framework.pagination import PageNumberPagination as BasePageNumberPagination
+from rest_framework.pagination import LimitOffsetPagination
```
Comment (rlucioni) [inline course_discovery/apps/api/pagination.py:2]: Can these stay on the same line so we're not importing from the same package twice? I'm surprised isort doesn't like this.

**Review by rlucioni — APPROVED** [review]
This is huge! Thanks.

**Review by bderusha — APPROVED** [review]
👍

File: course_discovery/apps/api/pagination.py (hunk 1 above)
Comment (clintonb) [reply course_discovery/apps/api/pagination.py:2]: The only way to allow that is to add an inline comment `# isort:skip`. Doing that however, would increase line length...requiring a comment to ignore that issue.

tl;dr it's not worth the hassle. Leaving as is.
